code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -274,17 +274,17 @@
 		to_chat(user, "<span class='danger'>This weapon is ineffective; it does no damage.</span>")
 		return 2
 
-	var/damage = O.force
-	if (O.damtype == PAIN)
-		damage = 0
-	if (O.damtype == STUN)
-		damage = (O.force / 8)
-	if(supernatural && istype(O,/obj/item/weapon/nullrod))
-		damage *= 2
-		purge = 3
-	adjustBruteLoss(damage)
-	if(O.edge || O.sharp)
-		adjustBleedTicks(damage)
+		var/damage = O.force
```

Comment:
All of this is now after a return and is unrelated to the PR itself.

Reply:
Missed it, thanks.

Reply:
This *should* be fixed (I straight up deleted the file and reverted to master's version), but Git is being a itself and refuses to recognize it, it seems.